src/main/components/PianoRollToolbar/VolumeSlider.tsx

```diff
@@ -33,7 +33,9 @@ export interface VolumeSliderProps {
 
 const _VolumeSlider: FC<VolumeSliderProps> = observer(({ trackId }) => {
   const rootStore = useStores()
-  const volume = rootStore.pianoRollStore.currentVolume
+  const volume = rootStore.song.tracks[trackId].getVolume(
+    rootStore.services.player.position
+  )
```

Comment:
Is this necessary change?

Reply:
I originally tried to add the volume slider to the track list, but my first attempt was quite buggy so I decided to drop it. I also wanted it to be global for the entire track—a percent between 0-100% that would be multiplyed with the tick spesific volume—but wasn't able to implement it easily. Can open an issue for this and drop the change :^)